main/guides/getting-started/index.md

```diff
@@ -32,7 +32,7 @@ Before getting started, there are some resources you might want to keep handy in
 
 Currently Agoric supports macOS and Linux (including [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about)). This tutorial is based on an installation of [Ubuntu 22.04 LTS](https://ubuntu.com/download/desktop). If you're using a different operating system, some variation may be required.
 
-## Installing Prerequisites
+## 1. Installing Prerequisites
```

Comment:
We used to have tools to detect broken links to section headings, but we don't any more.

 - https://github.com/Agoric/documentation/issues/1011

I'm not comfortable changing heading text until we can do that sort of checking.

Reply:
Agreed - we should probably delay changing heading text until we get some tooling in place...

Reply:
Agreed. I did not know headings were linked. I have removed the changes to headings in a new commit.